### ISSUE TITLE
How to Log Additional Variables in Tensorboard ?

### ISSUE THREAD

**Issue by minamo817** [espnet/espnet#5621]
Thanks for your continuous support.

I'm working on a multi-task enh model using ESPnet. I'm aware of the logging features provided by ESPnet (such as TensorBoard and loss curves). However, I'm looking for a way to log additional data, specifically intermediate variables within my loss-wrapper.

In my loss wrapper, the total loss is a combination of two separate losses (`loss = loss1 + loss2`). I'm interested in logging these individual components (`loss1` and `loss2`) for analysis. Is there a straightforward method in ESPnet to accomplish this, either via TensorBoard or some other way? or, is there any way to get “global_current_iter” ?

Any guidance or suggestions on how to solve this would be greatly appreciated.
